spec/unit/provider/service/systemd_spec.rb

```diff
@@ -134,6 +171,17 @@
       provider.disable
     end
   end
+  
+  describe "#mask" do
+    it "should run systemctl to disable and mask a service" do
+      provider = described_class.new(Puppet::Type.type(:service).new(:name => 'sshd.service'))
+      # I think that this is a symbol method due to the direct call from inside
+      # the provider. I'm not 100% sure.
+      provider.expects(:systemctl).with(:disable, 'sshd.service')
```

Comment:
We invoke `systemctl` with `"unmask"` as a string when unmasking (https://github.com/puppetlabs/puppet/pull/3684/files#diff-dafbe8211057bc10ff2cfe788a6c4ebbR70) but invoke `systemctl` with `:disable` as a symbol when disabling (https://github.com/puppetlabs/puppet/pull/3684/files#diff-dafbe8211057bc10ff2cfe788a6c4ebbR24); I believe that the underlying execution code converts all args to strings when executing. The end result is the same, but the code varies - nothing inherently harmful, just superficially inconsistent.